python/main.py

```diff
@@ -1,139 +1,176 @@
 import json
+import settings
+import ynab_client
 
-from flask import Flask, render_template, request, jsonify, redirect
+from flask import Flask, request, jsonify, redirect
 from datetime import datetime
 from dateutil.parser import parse
 from decimal import Decimal
 
 from pynYNAB.schema.budget import Transaction
 
-import settings
-import ynab_client
-
 app = Flask('Monzo to YNAB')
 app.config['DEBUG'] = settings.flask_debug
 
-if settings.sentry_dsn:
```

Comment:
My bad, don't think it's `.env.example`. That may have been something else. I meant `raven` isn't in `requirements.txt` :D